tests/search/test_LocalSearch.py

```diff
@@ -393,6 +394,39 @@ def test_intensify_can_improve_solution_further(rc208):
         assert_equal(ls.intensify(intensify_opt, cost_eval), intensify_opt)
 
 
+def test_intensify_can_swap_routes(ok_small):
+    """
+    Tests that the intensify method can improve a solution by swapping routes.
```

Comment:
This test relates to a bug (and its fix). Could you link back to the PR in this docstring? Something like "Asserts that the bug identified in #742 is fixed." is sufficient. That's helpful if ever we need to figure out why this test existed, and how it might need to be updated.